### PR TITLE
Issue #13100: Create tests from SeparatorWrap xdoc examples

### PR DESCRIPTION
**Pull request by stoyanK7**
Part of #13100

https://checkstyle.sourceforge.io/config_whitespace.html#SeparatorWrap

### CODE REVIEW

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/Example3.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+class Example3 {
+  String s;
+
+  int a,
+    b; // violation above '',' should be on a new line'
+
+  void foo(int a,
+                int b) { // violation above '',' should be on a new line'
+    int r
+      , t; // OK
```
Comment (romani) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/Example3.java:20]: Please do not copy `ok` comments.

**Review by romani — CHANGES_REQUESTED** [review]
https://github.com/checkstyle/checkstyle/issues/13213#issuecomment-1592017610

Items:

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/Example3.java (hunk 1 above)
Comment (stoyanK7) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/Example3.java:20]: I forgot to say. These comments I felt should be left to say. Don't know why. Please let me know if I really should remove them.

Comment (romani) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/Example3.java:20]: So there should be a reason defined in such comments.
Just "ok" is not ok

Comment (stoyanK7) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/Example3.java]: Done. Added explanation.

**Review by romani — APPROVED** [review]
Ok to merge

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/Example1.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  }
+
+  public void bar(int p
+                    , int q) { // violation '',' should be on the previous line'
+    if (s
+          .isEmpty()) { // violation ''.' should be on the previous line'
+    }
+  }
+}
+// xdoc section -- end
```
Comment (romani) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/Example1.java:26]: We need check for new line at end of example files

**Review by romani — CHANGES_REQUESTED** [review]
Items

**Review by nrmancuso — APPROVED** [review]
Ok to merge after https://github.com/checkstyle/checkstyle/pull/13228#discussion_r1232959966

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/Example1.java (hunk 2 above)
Comment (nrmancuso) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/Example1.java:26]: Issue created at https://github.com/checkstyle/checkstyle/issues/13244

Comment (stoyanK7) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/Example1.java]: Done. File is fixed.

**Review by romani — APPROVED** [review]
Ok to merge

**Pull request merged**